src/akamai/js/akamaiHD_flash.js

```diff
@@ -201,8 +203,30 @@ require("../../../html5-common/js/utils/constants.js");
      * @returns {boolean} True or false indicating success
      */
     this.setVideoUrl = function(url, encoding) {
-      /* Should be set to false if stream URL changes */
-      loaded = false;
+      var urlChanged = false;
+      controller = this.controller;
+
+      if (currentUrl.replace(/[\?&]_=[^&]+$/,'') != url)
+      {
+        currentUrl = url || "";
+
+        if (currentUrl.length > 0) 
+        {
+          currentUrl = currentUrl + (/\?/.test(currentUrl) ? "&" : "?") + "_=" + getRandomString();
+        }
+        urlChanged = true;
+        loaded = false;
+        url = "setVideoUrl("+currentUrl+")";
+      }
+      if (_.isEmpty(currentUrl)) 
+      {
+        controller.notify(controller.EVENTS.ERROR, { errorcode: 0 }); 
```

Comment:
I think this is same in OSMF but actually we might want to set the url to empty to unload a stream - we do this for ads in case the same ad is played twice.  Is it possible to handle empty string?